openquake/qa_tests_data/event_based_risk/occupants/expected/agg_curve.xml

```diff
@@ -12,13 +12,13 @@ xmlns:gml="http://www.opengis.net/gml"
     unit="people"
     >
         <poEs>
-            0.0384416 0.0231283 0.00508702 0.00239712 0.00139904 0.000499904 0.000399947 0.000199974 0.000199974 0.0
+            0.0384416 0.0240071 0.004888 0.00239712 0.00149888 0.000499904 0.000499904 0.000399947 0.000100017 0.0
```

Comment:
Even the occupants tests is indeed sensible to the epsilons, even if the report was not showing it. I have regenerated the report and checked manually that GMFs are the same and only the ordering of the epsilons is different.